apps/api/src/teams/event-registrations/event-registrations.controller.ts

```diff
@@ -30,9 +32,10 @@ export class TeamEventRegistrationsController {
   @CheckPolicies(ability => ability.can(Action.Read, TeamEvent))
   public async create(
     @Param('eventId', ParseIntPipe) eventId: number,
+    @Body() createTeamEventRegistrationDTO: CreateTeamEventRegistrationDTO,
```

Comment:
la loi est dure, mais c'est la loi
```suggestion
    @Body() createTeamEventRegistrationDto: CreateTeamEventRegistrationDto,
```